src/components/site/NavLegal.js

```diff
@@ -17,24 +17,32 @@ const HeaderNav = () => {
         <Link
           className="mr-11 md:mr-0 md:ml-9 transform transition-all duration-300 ease-in-out hover:opacity-50"
           to={siteMetadata.element}
+          name="Element"
```

Comment:
Looks like name is deprecated, can be replaced with an id if we need it
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a